js/modules.js

```diff
@@ -15,52 +15,79 @@ CORE.create_module("email-nav", function (sb) {
 CORE.create_module("email-catalog", function (sb) {
     var emails;
 
+    var openEmail = function(e) {
+        sb.hide();
+        sb.notify({
+            type : "open-email",
+            data : e.currentTarget.getAttribute("data-email-id")
+        });
+    }
+    var selectEmailCheckbox = function(e) {
+        e.stopPropagation();
+        sb.notify({
+            type : "select-email-checkbox",
+            data : e.currentTarget.getAttribute("data-email-id")
+        });
+    }
+    var markSelectedEmailsAsRead = function() {
+        sb.notify({ 
+            type : "mark-selected-emails-as-read"
+        });
+    }
+
+    var fetchEmails = function(data) {
+        emails = data;
+        sb.template(emails);
```

Comment:
Ohh i see. I'm cool with it. If you wanna write it cleaner, i guess you could do something like `sb.template(emails=data)`. That should work but IMO that looks dirty. I'd leave it as is.